openupgrade_scripts/scripts/website_payment/16.0.1.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.delete_records_safely_by_xml_id(
+        env,
+        [
+            "website_payment.mail_template_donation",
+        ],
+    )
+    openupgrade.load_data(
+        env.cr, "website_payment", "migrations/16.0.1.0/noupdate_changes.xml"
```

Comment:
I think that the correct location is `"16.0.1.0/noupdate_changes.xml"` and not under the `migrations` directory.